include/CL/cl.h

```diff
@@ -1375,6 +1375,18 @@ clEnqueueTask(cl_command_queue  /* command_queue */,
               const cl_event *  /* event_wait_list */,
               cl_event *        /* event */) CL_EXT_SUFFIX__VERSION_1_2_DEPRECATED;
 
+/* Poclcc OpenCL APIs */
+extern CL_API_ENTRY cl_int CL_API_CALL
+clExportPoclccBinary(cl_program /* program */,
```

Comment:
These functions are just helpers.

I don't know how you think the binary format will be used... I chosed to have a binary format with (possibly) several device in it. But the _clGetProgramInfo_ gives a tab of devices. So I made those functions to put devices together and to add few other info (pocl identifier, version number).

Maybe I won't be usefull to have binaries for severals devices in the binary format. In this case, those function wouldn't have any purpose. And we would just have to add the version number in the device format (as it is not present at the moment).